src/JsonSchema/SchemaFactory.php

```diff
@@ -253,6 +254,14 @@ private function buildDefinitionName(string $className, string $format = 'json',
         return $name;
     }
 
+    private function encodeDefinitionName(string $name): string
+    {
+        $entities = ['%2F', '%7E'];
+        $replacements = ['~1', '~0'];
```

Comment:
Why is it necessary? IIRC only `~` and `/` must be encoded, not there URL encoded versions. Or am I missing something?

Reply:
Please have a look to https://github.com/api-platform/core/issues/1969#issuecomment-392033381